azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/service/calling/CallingService.kt

```diff
@@ -122,7 +129,14 @@ internal class CallingService(
         coroutineScope.launch {
             callingSDKWrapper.getCallingStateWrapperSharedFlow().collect {
                 val callStateError = when (it.callEndReason) {
-                    CALL_END_REASON_SUCCESS, CALL_END_REASON_CANCELED, CALL_END_REASON_DECLINED -> null
+                    CALL_END_REASON_SUCCESS -> {
```

Comment:
I feel with kotlin this would be inline if would read better.

CALL_END_REASON_SUCCESS -> if (isEvicted(it)) CallStateError(...) else null

something about an else block with just "null" in it, or leaving out the returns just feels off here.

Reply:
@ahammer I have moved this into getCallStateError() function